DataImport.Common.Tests/ConstantsTests.cs

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using IngestionLogLevelConstants = DataImport.Common.Constants.IngestionLogLevel;
+
+namespace DataImport.Common.Tests
+{
+    [TestFixture]
+    internal class ConstantsTests
```

Comment:
@jagudelo-gap 
Please rename to LogLevelTests ( what ever the name of the log level class) !!

Reply:
@CSR2017 
Done